Makefile

```diff
@@ -206,7 +233,9 @@ fjalar/valgrind/Makefile.am: ../fjalar/auto-everything.sh
 	ln -nsf ../fjalar fjalar
 	touch $@
 
-fjalar/valgrind/Makefile.in: fjalar/valgrind/Makefile.am
```

Comment:
What is the purpose of this change?  I would have thought the before and after versions behaved the same (but the before seems more concise).